Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -182,6 +179,7 @@ class BrowserViewController: UIViewController {
     deinit {
         NSNotificationCenter.defaultCenter().removeObserver(self, name: UIApplicationWillChangeStatusBarFrameNotification, object: nil)
         NSNotificationCenter.defaultCenter().removeObserver(self, name: BookmarkStatusChangedNotification, object: nil)
+        NSNotificationCenter.defaultCenter().removeObserver(self, name: "ThemeImageChanged", object: nil)
```

Comment:
Looks like you're listening to the theme changed events from within the LightweightThemeManager so I don't think you need this here.